content/blog-posts/2023-04-01-excelbpf/index.md

```diff
@@ -0,0 +1,28 @@
+
+Looking at where eBPF 1.0 has transformed the world shows us some of the potential excelBPF has for the future. In the cloud native world, [eBPF has unlocked innovation](https://www.infoworld.com/article/3689690/how-ebpf-unlocks-cloud-native-innovation.html) by shortening production feedback loops, extended functionality with a safety belt, provided new building blocks while still leveraging existing tooling, allowing for on the fly upgrades, making data processing faster with lower resource consumption, and creating better abstractions for observability and security. excelBPF is bringing these benefits outside the IT department. Spreadsheets no longer need to be held back by decades old macros.
+
+“With eBPF 1.0, [Cilium](cilium.io) was able to transform the cloud native infrastructure landscape in areas as diverse as modernizing telecommunications networks, bringing retail stores online, and moving money through markets. Our end users are excited at what excelBPF will bring to the rest of the organization. Just as prompt engineering has taken off with the launch of large language models, I think we soon see job advertisements for excelBPF engineers who are able to change spreadsheets from individual cells to hives of activity.” said Thomas Graf co-founder of the Cilium project.
+
+Early versions of excelBPF were put in the hands of the community, and feedback has been impressive. “Forget about ChatGTP, FTX and SVB, excelBPF has completely stopped the press for me. As someone striving to be the living embodiment of "Operational Excellence", excelBPF is the only tool I need in my arsenal. Everything one needs spanning FinOps, HROps, SalesOps, RevOps, BizOps, CloudOps and even OpsOps is now just an excelBPF add-on model away. I finally have some time to pretend like I'm going surfing again!” said Christiaan Kuun, Head of Operations at Isovalent.
+
+With money flowing this quickly through excelBPF, you could easily guess that the funding for companies is just behind it and you wouldn’t be wrong. “After seeing how fast Isovalent changed the cloud native landscape with eBPF 1.0, we have an open checkbook for companies looking to build around excelBPF. I'm literally just waiting for those excelBPF financial models to hit my inbox.” said Martanique Casandro, Partner at Redwood Tree Venturez (r17z).
+
+The [eBPF Landscape](https://ebpf.io/applications/) is rapidly evolving and we are excited to see the first excelBPF powered applications be added to the landscape. Be sure to check back regularly to see how the space is evolving. If you want to catch up on eBPF 1.0 before diving into excelBPF, check out [what is eBPF](https://ebpf.io/what-is-ebpf/), the [getting started page](https://ebpf.io/get-started/), and our [interactive eBPF labs](https://play.instruqt.com/embed/isovalent/tracks/ebpf-getting-started?token=em_9nxLzhlV41gb3rKM&show_challenges=true).
```

Comment:
Link to the eCHO newsletter as well? Or it's easy to find from the get-started page probably

Reply:
```suggestion
The [eBPF Landscape](https://ebpf.io/applications/) is rapidly evolving and we are excited to see the first **excelBPF™** powered applications be added to the landscape. Be sure to check back regularly to see how the space is evolving. If you want to catch up on eBPF 1.0 before diving into excelBPF, check out [what is eBPF](https://ebpf.io/what-is-ebpf/), the [getting started page](https://ebpf.io/get-started/), our [interactive eBPF labs](https://play.instruqt.com/embed/isovalent/tracks/ebpf-getting-started?token=em_9nxLzhlV41gb3rKM&show_challenges=true), and subscribe to [eCHO News](https://ebpf.io/newsletter/) to keep up on all the updates from the eBPF community.
```